src/i18n/rpi-imager_sk.ts

```diff
@@ -648,7 +648,7 @@
     <message>
         <location filename="../main.qml" line="500"/>
         <source>[ All ]</source>
-        <translation type="unfinished"></translation>
+        <translation>[ Všetky ]</translation>
```

Comment:
I'm particularly impressed you translated this string - it's often omitted by translators.